### DISCUSSION TITLE
imputation_type = 'iterative' not working

### DISCUSSION THREAD

**Discussion by b00nw33** [pycaret/pycaret#3754] [Q&A]
I am trying out using iterative imputation. I get the typical tabular output from running the following code.
```
classf = setup(data = df1, target = targetcol, imputation_type = 'iterative')
```

However, the following line would freeze up the session.
```
best = compare_models()
```

Please advise if I am doing something wrong. Thank you.

**Comment by tvdboom** [comment]
The fact that the code is freezing indicates that the issue is probably related to your data or your machine. You can try the same code using a very small dataset. If that still doesn't work, open an issue with reproducible code and data.
